src/server/dash-server.ts

```diff
@@ -185,15 +182,8 @@ export class DashServer {
             if (!review) {
               continue;
             }
-            const result = {
-              author: '',
-              createdAt: Date.parse(review.createdAt),
-              reviewState: review.state,
-            };
-            if (review.author && review.author.__typename === 'User') {
-              result.author = review.author.login;
-            }
-            object.reviews.push(result);
+
+            object.reviews.push(reviewFieldsToResult(review));
```

Comment:
Could you name object to something more descriptive.

Reply:
Done.